src/main/java/org/dependencytrack/model/ConfigPropertyConstants.java

```diff
@@ -45,6 +45,10 @@ public enum ConfigPropertyConstants {
     JIRA_USERNAME("integrations", "jira.username", null, PropertyType.STRING, "The optional username to authenticate with when creating an Jira issue", ConfigPropertyAccessMode.READ_WRITE),
     JIRA_PASSWORD("integrations", "jira.password", null, PropertyType.ENCRYPTEDSTRING, "The optional password for the username used for authentication", ConfigPropertyAccessMode.READ_WRITE),
     MAINTENANCE_METRICS_RETENTION_DAYS("maintenance", "metrics.retention.days", "90", PropertyType.INTEGER, "Number of days to retain metrics data for", ConfigPropertyAccessMode.READ_WRITE),
+    MAINTENANCE_PROJECTS_RETENTION_ENABLE("maintenance", "projects.retention.enable", "false", PropertyType.BOOLEAN, "Enable Inactive Project Deletion", ConfigPropertyAccessMode.READ_WRITE),
```

Comment:
Instead of introducing a separate flag, you could treat `MAINTENANCE_PROJECTS_RETENTION_TYPE != ('AGE', 'VERSIONS')` as disabled. So `null`, `foo`, `123`, etc. would all qualify as "not enabled". Defaulting the property to `null` disables the feature initially.